### DISCUSSION TITLE
How to increase inference speed?

### DISCUSSION THREAD

**Discussion by carlosseda** [oobabooga/text-generation-webui#1136] [Q&A]
I am using the model anon8231489123/vicuna-13b-GPTQ-4bit-128g on an Nvidia 4070 Ti. Through the API, I am achieving this inference speed:

Output generated in 27.89 seconds (10.75 tokens/s, 300 tokens, context 485, seed 874383589)

Due to the task I want to perform, I cannot reduce the context or the output tokens. Is there any way to increase the output to 300 tokens/s? For example, by using an A100?

Thanks.

**Comment by BetaDoggo** [comment]
I believe 4bit with gptq is slower than fp16 currently. If you were able to load the full sized model you could probably make it go a bit faster.

**Comment by ginieman** [comment]
I have 4070Ti too. If I use oobabooga local UI, it takes a few seconds to get response. However, if I use API via TavernAI/SillyAI, it takes around 30s to get response. It seems API mode can be further optimized.  Does anyone have any solution?

**Reply by ghost** [reply]
I have 2060 with 12GB VRAM and I've noticed the same.. Local UI of oobabooga barely takes any time but if I use TavernAI, it is around 20-30s wait. I saw someone commenting that it is due to some update like week ago but there wasn't any help there either..
It's a shame though, I really like this project but I think only solution atm is just to use different thing with TavernAI.. KoboldAI for example. I haven't tried though if it works faster or better
